docusaurus/docs/learn/core-concepts/services/overview.mdx

```diff
@@ -143,3 +143,339 @@ This strategy drives costs in the same way as the `smartrouting` strategy. Howev
 ##### `random`
 This strategy does not change terminator weights. It does simple random selection across all terminators of the highest precedence. 
 
+## Practical Service Hosting
+
+### Edge Router Tunneler Hosting
+
+#### Single Application Endpoint
+When hosting services with the edge router tunneler (ER/T) combination you'll need to use a service configurations. We're going
+to start off simply, with one service endpoint and build up from there.
+
+Our application server is going to be on a local subnet at IP 192.168.3.136, port 8080. For our `test` service, we make
+ and initial service configuration using the CLI as follows:
+
+```
+ziti edge create config test-host-config host.v2 '
+{
+    "terminators" : [
+        {
+            "address": "192.168.3.136",
+            "port" : 8080,
+            "protocol": "tcp"
+        },
+    ]
+}
+'
+
+ziti edge create service test -c test-host-config --terminator-strategy smartrouting
+
+ziti edge create edge-router edge-router-1 --tunneler-enabled
+ziti edge create edge-router edge-router-2 --tunneler-enabled
+
+# skipping router enrollment steps
+
+ziti edge update identity edge-router-1 --role-attributes 'test-host'
+ziti edge update identity edge-router-2 --role-attributes 'test-host'
+
+ziti edge create service-edge-router-policy test-serp --service-roles '@test' --edge-router-roles '#all'
+ziti edge create service-policy test-bind Bind --service-roles '@test' --identity-roles '#test-host'
+```
+
+This will provide basic access to the service with one or many ER/Ts. All edge routers are hitting the same endpoint,
+so they don't need any customized configurations. Each ER/T hosting the service will create a terminator for the service
+and traffic will get load-balanced across them.
```

Comment:
> traffic will get load-balanced across them.

Is it more accurate to state that the traffic will be routed based on how it was setup? AKA Primary/Secondary is not "load-balanced", right?

Reply:
Clarified.